src/nat/llm/huggingface_llm.py

```
@@ -0,0 +1,289 @@
+
+    def invoke(self, messages, **kwargs):
+        """Synchronous invoke - wraps async version."""
+        import asyncio
+        try:
+            loop = asyncio.get_event_loop()
+        except RuntimeError:
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
+        return loop.run_until_complete(self.ainvoke(messages, **kwargs))
+
+    async def ainvoke(self, messages, **kwargs):
```

Ruff (0.14.8)

Unused method argument: `kwargs`

(ARG002)